include/linker/ramfunc.ld

```diff
@@ -0,0 +1,19 @@
+SECTION_DATA_PROLOGUE(.ramfunc,,)
+{
+#ifdef CONFIG_MPU_REQUIRES_POWER_OF_TWO_ALIGNMENT
+	. = ALIGN( 1 << LOG2CEIL(__ramfunc_ram_end - __ramfunc_ram_start));
+#endif
+	__ramfunc_ram_start = .;
+	KERNEL_INPUT_SECTION(.ramfunc)
+	KERNEL_INPUT_SECTION(".ramfunc.*")
+#ifdef CPU_HAS_ARM_MPU
+	. = ALIGN(32);
+#endif
+#ifdef CONFIG_MPU_REQUIRES_POWER_OF_TWO_ALIGNMENT
+	. = ALIGN( 1 << LOG2CEIL(__ramfunc_ram_end - __ramfunc_ram_start));
+#endif
+	__ramfunc_ram_end = .;
+	. = ALIGN(32);
+} GROUP_DATA_LINK_IN(RAMABLE_REGION, ROMABLE_REGION)
+
+__ramfunc_rom_start = LOADADDR(.ramfunc);
```

Comment:
Something wrong with the new line at the end of the file?

Reply:
No, It's my mistake.